json-path/src/main/java/com/jayway/jsonpath/internal/function/sequence/AbstractSequenceAggregation.java

```diff
@@ -37,7 +37,7 @@ public Object invoke(String currentPath, PathRef parent, Object model, Evaluatio
                 }
             }
         }
-        throw new JsonPathException("Aggregation function attempted to calculate value using empty array");
+        throw new JsonPathException("Aggregation function attempted to calculate value using non array");
```

Comment:
I guess it should go as a separate commit or even PR

Reply:
Thanks. Reverted